torchdynamo/allowed_functions.py

```diff
@@ -116,12 +116,17 @@ def _allowed_function_ids():
 
     def _is_allowed_module_prefix(obj):
         allowed_modules = ("torch", "math")
+        disallowed_modules = "torch.optim."
         allowed_modules_dot = tuple([x + "." for x in allowed_modules])
         module = inspect.getmodule(obj)
         if module is None:
             return False
 
         mod_name = module.__name__
+
+        if mod_name.startswith(disallowed_modules):
```

Comment:
nit - move `allowed_modules` and `allowed_modules_dot` lines below these checks?